run_unittests.py

```diff
@@ -4347,7 +4347,7 @@ def test_summary(self):
                          A list: string
                                  1
                                  True
-                     empty list: 
+                     empty list:
```

Comment:
That's unrelated change, and I'm not sure it's correct. Was it failing the CI here?

Reply:
No, I have my editor set up to trim trailing whitespace, so every time I save this it gets deleted. Is there a good reason that the summary code prints the whiltespace, or could we just remove it and not have it in the test either?

Reply:
It's purely an implementation detail, feel free to fix it ;-)

Reply:
Did it there, while at it: https://github.com/mesonbuild/meson/pull/6692